html/dom/interfaces.https.html

```diff
@@ -33,11 +33,19 @@ <h1>HTML IDL tests</h1>
   return input;
 }
 
+function getFormDataEvent() {
+  let formDataEvent = null;
+  let form = document.createElement("form");
+  form.addEventListener('formdata', e => { formDataEvent = e });
+  new FormData(form);
+  return formDataEvent;
+}
+
 const waitForLoad = new Promise(resolve => { addEventListener('load', resolve); })
 
 idl_test(
   ['html'],
-  ['SVG', 'cssom', 'touch-events', 'uievents', 'dom'],
+  ['SVG', 'cssom', 'touch-events', 'uievents', 'dom', 'xhr'],
```

Comment:
Isn't the interface added to HTML?

Reply:
Yes.  However 'xhr' is necessary for FormData interface dependency.